docs/elements.md

```diff
@@ -972,12 +973,39 @@ The `focus` attribute defaults to `"false"`. Setting this to true will cause the
 #### See also
 
 - [`pl-file-upload` to receive files as a submission](#pl-file-upload-element)
+- [`pl-file-preview` to display previously submitted files](#pl-file-preview)
 - [`pl-external-grader-results` to include output from autograded code](#pl-external-grader-results)
 - [`pl-code` to display blocks of code with syntax highlighting](#pl-code-element)
 - [`pl-string-input` for receiving a single string value](#pl-string-input-element)
 
 ---
 
+### `pl-file-preview` element
+
+Provides an in-browser preview of a file submitted by a student as part of an external grading system. Shows the submitted
+file in the corresponding submission panel. Usually used in conjunction with some combination of `pl-file-editor`,
+`pl-file-upload`, and `pl-external-grader-results` elements.
```

Comment:
It may be misleading to list `pl-external-grader-results` here, since it is a companion to `pl-file-preview`, while the other elements are file submission elements. You could list other elements that submit files, like `pl-order-blocks` and `pl-rich-text-editor`, and list `pl-external-grader-results in a separate sentence.